assets/fonts/fonts.css

```diff
@@ -1,17 +1,61 @@
+/* SANS */
 
-/* Demi */
+/* Regular */
 @font-face {
   font-display: fallback;
-  font-family: "Avenir Next Webfont";
-  src: url("249228f0-61ac-40cc-a5a5-5609c9816e3f.woff2") format("woff2"),
-    url("efba18ed-80cc-49c4-997a-fbb140739d19.woff") format("woff");
+  font-family: "Noto Sans";
+  src: url("NotoSans-Regular-webfont-western-latin.woff2") format("woff2");
+}
+
+/* Bold */
+@font-face {
+  font-display: fallback;
+  font-family: "Noto Sans";
+  src: url("NotoSans-Bold-webfont-western-latin.woff2") format("woff2");
+  font-weight: 700;
```

Comment:
Consider whether `font-weight` (here and on line 24) should now be set to `bold` (or `800`), now we're not using a demi bold. (This would require corresponding changes in the patterns' CSS.)

Reply:
Confirmed with design that all bold-like weights are now one.

To be expressed by `font-weight: bold;`